applications/volume_rendering/python/CMakeLists.txt

```diff
@@ -37,9 +37,8 @@ if(HOLOHUB_DOWNLOAD_DATASETS)
 endif()
```

Comment:
I see an internal volume_rendering C++ coverage report in CDash, looks like coverage is being reported as missing for third party libraries under `_deps`.

How can a developer tell gcov to ignore certain directories or projects in the coverage report under the current `./holohub test` proposal here? Is it possible? Not a blocker